### ISSUE TITLE
Some wind speed maximums less than wind speed averages

### ISSUE THREAD

**Issue by wmay** [ASRCsoft/atmoschem.datasets#95]
This seems to happen a lot in the 2013-2015 WFMS data.

**Comment by wmay** [comment]
This is just a processing error. The numbers are vector averages of the 1-minute maximums, instead of hourly maximums.

**Comment by RB-ASRC** [comment]
Thanks for sorting this out.  I think when we went to two anemometers, the idea was to choose the higher of the two to report for the 1-hour average. We also logged instantaneous max wind speed for one of the anemometers in table 1 and both in table 3. Later we logged max for both in table 1.  My guess is that when only sensor 1 max wind speed was reported  in table 1 and sensor 2 had a 1-min average higher than sensor 1 max, max<1min ave.
